_content/lorawan/lorawan_specs.md

```diff
@@ -0,0 +1,20 @@
+---
+title: LoRaWAN Specifications
+---
+
+# LoRaWAN Specifications
+
+The LoRaWAN™ specification documents are developed and maintained by the LoRa Alliance: an open association of collaborating members. To fully define the LoRaWAN protocol, and to ensure interoperability among devices and networks, the LoRa Alliance develops and maintains the following documents:
+
+* Core LoRaWAN Specification
+* Regional Parameters
+* Certification Program
+* Back-End Interfaces
+
+While these specifications define the technical implementation, they do not define any commercial model or type of deployment (public, shared, private, enterprise) and so offer the industry the freedom to innovate and differentiate in how a LoRaWAN system is implemented.
+
+The specification defines the and so provides seamless interoperability between manufacturers, as demonstrated via the device certification program.
```

Comment:
> The specification defines the and

Missing word?

Reply:
Try this for brevity..
defines the and so provides -> enables